packages/edge-gateway/wrangler.toml

```diff
@@ -105,11 +103,17 @@ name = "PUBLIC_RACE_STATUS_CODE"
 # Test!
 [env.test]
 workers_dev = true
-kv_namespaces = [{ binding = "DENYLIST" }]
+
+[[env.production.services]]
```

Comment:
We need staging and production. And please put it in order with the other configs for each environment